Tests/Controller/HelperControllerTest.php

```diff
@@ -460,7 +461,17 @@ public function testSetObjectFieldValueActionWithViolations()
             new ConstraintViolation('error2', null, array(), null, 'enabled', null),
         ));
 
-        $validator = $this->getMock('Symfony\Component\Validator\ValidatorInterface');
+        if (version_compare(Kernel::VERSION, '2.5', '<')) {
```

Comment:
Actually, I copied this check from  #2176. But I recognize the test should check both API versions with every Symfony version.